lib/LicenseTermsList/LicenseTermsList.js

```diff
@@ -35,24 +38,32 @@ export default class LicenseTermsList extends React.Component {
     const note = get(this.props.license, ['customProperties', term.name, '0', 'note']);
 
     return (
-      <Layout className="padding-top-gutter" key={index}>
-        <Row>
-          <Col xs={12} md={4}>
-            {term.description ? <InfoPopover data-test-info-popover content={term.description} /> : null}
-            <strong data-test-term-label={term.name}>{term.label}</strong>
-          </Col>
-          <Col xs={12} md={8}>
-            <span data-test-term-value={term.name}>{value}</span>
-          </Col>
-        </Row>
-        { note ? (
+      <Layout key={index}>
```

Comment:
I don't think this `Layout` element is necessary here. `Layout` only does something if you pass it a `className` prop.